rest-catalog/quarkus-rest-catalog/src/main/resources/application.properties

```diff
@@ -52,18 +52,20 @@ quarkus.log.category."io.quarkus.http.access-log".level=${HTTP_ACCESS_LOG_LEVEL:
 quarkus.http.port=19130
 quarkus.http.test-port=0
 quarkus.http.access-log.enabled=true
-# fixed at buildtime
-quarkus.resteasy.path=/
-quarkus.resteasy.gzip.enabled=true
```

Comment:
resteasy-reactive does not have an equivalent property, but rather relies on `quarkus.http.enable-compression`. I manually verified that responses are being compressed if the user agent includes an `Accept-Encoding: gzip` header in the request.

Reply:
okay